### ISSUE TITLE
Contacts auto sync function

### ISSUE THREAD

**Issue by UKPhil** [ubports/ubuntu-touch#331]
I have my gmail contacts linked to my UBPorts phone. I added a new contact today from a call I received. I then had to go into contacts and sync this with my online account. 

Is there a setting (or can we have a feature) to have the sync performed automatically when a change is made either online or locally?

**Comment by mardy** [comment]
Hi! AFAIK, there isn't such a feature, but I think we should add it. At the moment in the Contacts app there is not even a list of what are the enabled contacts sources.
We should add it, and add a sync button as well (or a page with sync options).

One thing I have observed, but I'm not 100% sure about, is that a sync is triggered when you connect to WLAN (not mobile data connection!). You could try disconnecting and reconnecting to a WLAN, and see if that works.

**Comment by tomoqv** [comment]
Another problem is that the contacts app defaults to the "Personal" address book. I often forget to change to Google when adding a new contact, and unfortunately there seems to be no way to edit which address book the contact is stored in once it has been saved on the phone.

**Comment by UKPhil** [comment]
I totally agree  I think the contacts app needs a bit of work on it to bring it up to speed. 

Also this now needs to be moved from a question to an issue as it is affecting multiple users.
